include/abb_librws/rws_common.h

```diff
@@ -660,6 +660,194 @@ struct SystemConstants
   };
 };
 
+/**
+ * \brief A class for a tri value bool.
+ */
+class TriBool
+{
+public:
+  /**
+   * \brief An enum for the different accepted values.
+   */
+  enum Values
```

Comment:
Why the plural? I know there are multiple valid values, but it gets weird when you declare a variable of this type: `Values value`. It really just holds a single value, does it not?

Reply:
That's how I do it in general, and I think it's logical to have the type as plural (since there are several accepted values) and then have singular in the instance. But I guess it's a bit subjective :)

Reply:
That's how I usually do it, and I think it's logical to have the type as plural (since there are several accepted values) and then have singular in the instance. But I guess it's a bit subjective :)